docs/reference/announcements-release-notes/release-policy.md

```diff
@@ -137,10 +137,25 @@ vast majority of dependencies Camunda _only_ applies patch updates.
 
 However, certain dependencies used by Camunda 8 may have a shorter maintenance policy than Camunda itself. Camunda may adopt a different update policy for these dependencies, as listed below.
 
-### Spring Boot
+### Spring Boot/Framework/Security Update Policies
 
-The **Camunda Orchestration Cluster** is a Spring Boot application and leverages Spring Boot to execute fundamental functionality such as application configuration, REST infrastructure (including security), production ready features, etc.
+Spring has a [different maintenance window](https://spring.io/projects/spring-boot#support) than Camunda for its open-source software (OSS) offering.
+Camunda addresses this with a dedicated Spring Boot/Framework/Security Update Policy for the Camunda Orchestration Cluster and for client artifacts.
 
-However, Spring Boot has a shorter maintenance window than Camunda for its open-source software (OSS) offering. [Versions are only supported for 13 months](https://spring.io/projects/spring-boot#support), versus Camunda's 18 months. To circumvent this, the **Orchestration Cluster** patch releases also update Spring Boot minor versions, such that the latest patch release of these components uses a supported Spring version.
+Note: The version of the Spring Framework and Spring Security as well as other dependencies managed via Spring Boot (see [Managed Dependency Coordinates](https://docs.spring.io/spring-boot/appendix/dependency-versions/coordinates.html)) are covered by the same policy.
 
-As for libraries and SDKs meant to be included in third-party applications, Camunda follows a best effort policy to balance compatibility and secure Spring-dependent libraries.
+#### Camunda Orchestration Cluster Spring Boot/Framework/Security Update Policy
```

Comment:
This header is already under `Dependency maintenance policies > Spring Boot/Framework/Security updates`, so I would suggest we go with "Orchestration Cluster".